R/formatter.r

```diff
@@ -314,3 +314,34 @@ unit_format <- function(unit = "m", scale = 1, sep = " ", ...) {
     paste(comma(x * scale, ...), unit, sep = sep)
   }
 }
+
+#' Formatted dates and times.
+#'
+#' @param format Date/time format using standard POSIX specification.  See
+#'  [strptime()] for possible formats.
+#' @param tz a time zone name, see [timezones()]. Defaults
+#'  to UTC
+#' @export
```

Comment:
Can has examples?

Reply:
👍 done. Also removed the stop_if from tests because I realized it was unnecessary, as a suggests it is loaded.